doc/quickstart-guide.md

```diff
@@ -0,0 +1,36 @@
+# Install the 3scale operator through Operator Lifecycle Manager (OLM).
+
+You will need access to an OpenShift Container Platform 4.1 cluster.
+
+Procedure
+1. In the OpenShift Container Platform console, log in using an account with administrator privileges.
+1. Create new project `operator-test` in *Projects > Create Project*.
+1. Click *Catalog > OperatorHub*.
+1. In the Filter by keyword box, type 3scale operator to find the 3scale operator.
+1. Click the 3scale operator. Information about the Operator is displayed.
+1. Click *Install*. The Create Operator Subscription page opens.
+1. On the *Create Operator Subscription* page, accept all of the default selections and click Subscribe.
+1. After the subscription *upgrade status* is shown as *Up to date*, click *Catalog > Installed Operators* to verify that the 3scale operator ClusterServiceVersion (CSV) is displayed and its Status ultimately resolves to _InstallSucceeded_ in the `operator-test` project.
+
+# Deploying 3scale using the operator
```

Comment:
If this doc is upstream-only it is ok. The moment this is moved to productized doc we should not forget about mentioning the creation of the redhat authenticated registry secret as a previous step

Reply:
Sure. 

Anyway, official productized doc does not live in this GH repo. What I mean is that it is not mandatory we have specific documentation for productized releases.

Reply:
:+1: I agree